### DISCUSSION TITLE
Add option to only show spawner hologram a certain amount of time

### DISCUSSION THREAD

**Discussion by RealScarmo** [OmerBenGera/WildStacker#135] [Denied Feature Requests] [answered]
I would appreciate if an option was added to the config where you can choose to only show spawner holograms for x amount of time after an entity has spawned from the spawner.

**Comment by OmerBenGera** [comment, marked as answer]
It's really similar to the floating names feature. Does it fit your needs?

**Reply by RealScarmo** [reply]
It does. Didn’t know about this option.
